2025-05-18 12:22:32,611 INFO    MainThread:3612948 [wandb_setup.py:_flush():76] Current SDK version is 0.16.4
2025-05-18 12:22:32,611 INFO    MainThread:3612948 [wandb_setup.py:_flush():76] Configure stats pid to 3612948
2025-05-18 12:22:32,611 INFO    MainThread:3612948 [wandb_setup.py:_flush():76] Loading settings from /home/sjwalker/.config/wandb/settings
2025-05-18 12:22:32,611 INFO    MainThread:3612948 [wandb_setup.py:_flush():76] Loading settings from /home/sjwalker/nd0821-c2-build-model-workflow-exercises-4/lesson-2-data-exploration-and-preparation/exercises/exercise_4/starter/wandb/settings
2025-05-18 12:22:32,611 INFO    MainThread:3612948 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2025-05-18 12:22:32,611 INFO    MainThread:3612948 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2025-05-18 12:22:32,611 INFO    MainThread:3612948 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2025-05-18 12:22:32,611 INFO    MainThread:3612948 [wandb_init.py:_log_setup():526] Logging user logs to /home/sjwalker/nd0821-c2-build-model-workflow-exercises-4/lesson-2-data-exploration-and-preparation/exercises/exercise_4/starter/wandb/run-20250518_122232-el5in8b2/logs/debug.log
2025-05-18 12:22:32,611 INFO    MainThread:3612948 [wandb_init.py:_log_setup():527] Logging internal logs to /home/sjwalker/nd0821-c2-build-model-workflow-exercises-4/lesson-2-data-exploration-and-preparation/exercises/exercise_4/starter/wandb/run-20250518_122232-el5in8b2/logs/debug-internal.log
2025-05-18 12:22:32,611 INFO    MainThread:3612948 [wandb_init.py:init():566] calling init triggers
2025-05-18 12:22:32,612 INFO    MainThread:3612948 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2025-05-18 12:22:32,612 INFO    MainThread:3612948 [wandb_init.py:init():591] re-initializing run, found existing run on stack: sbxa1flw
2025-05-18 12:22:32,613 INFO    MainThread:3612948 [wandb_run.py:_finish():1970] finishing run swal706-western-governors-university/exercise_4/sbxa1flw
2025-05-18 12:22:32,686 INFO    MainThread:3612948 [jupyter.py:save_history():474] saving 9 cells to _session_history.ipynb
2025-05-18 12:22:32,687 INFO    MainThread:3612948 [wandb_run.py:_config_callback():1343] config_cb ('_wandb', 'session_history') code/_session_history.ipynb None
2025-05-18 12:22:32,691 INFO    MainThread:3612948 [jupyter.py:_save_ipynb():384] looking for notebook: EDA.ipynb
2025-05-18 12:22:32,822 INFO    MainThread:3612948 [wandb_init.py:_jupyter_teardown():453] saved code and history: <Artifact source-exercise_4-EDA.ipynb>
2025-05-18 12:22:32,822 INFO    MainThread:3612948 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2025-05-18 12:22:32,822 INFO    MainThread:3612948 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2025-05-18 12:22:32,822 INFO    MainThread:3612948 [wandb_run.py:_restore():2193] restore
2025-05-18 12:22:32,823 INFO    MainThread:3612948 [wandb_run.py:_restore():2199] restore done
2025-05-18 12:22:37,492 INFO    MainThread:3612948 [wandb_run.py:_footer_history_summary_info():3851] rendering history
2025-05-18 12:22:37,492 INFO    MainThread:3612948 [wandb_run.py:_footer_history_summary_info():3883] rendering summary
2025-05-18 12:22:37,494 INFO    MainThread:3612948 [wandb_run.py:_footer_sync_info():3810] logging synced files
2025-05-18 12:22:37,639 INFO    MainThread:3612948 [wandb_init.py:init():616] starting backend
2025-05-18 12:22:37,639 INFO    MainThread:3612948 [wandb_init.py:init():620] setting up manager
2025-05-18 12:22:37,641 INFO    MainThread:3612948 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-05-18 12:22:37,641 INFO    MainThread:3612948 [wandb_init.py:init():628] backend started and connected
2025-05-18 12:22:37,644 INFO    MainThread:3612948 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2025-05-18 12:22:37,644 INFO    MainThread:3612948 [wandb_init.py:init():720] updated telemetry
2025-05-18 12:22:37,650 INFO    MainThread:3612948 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2025-05-18 12:22:38,773 INFO    MainThread:3612948 [wandb_run.py:_on_init():2262] communicating current version
2025-05-18 12:22:38,841 INFO    MainThread:3612948 [wandb_run.py:_on_init():2271] got version response upgrade_message: "wandb version 0.19.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2025-05-18 12:22:38,841 INFO    MainThread:3612948 [wandb_init.py:init():804] starting run threads in backend
2025-05-18 12:22:42,138 INFO    MainThread:3612948 [wandb_run.py:_console_start():2241] atexit reg
2025-05-18 12:22:42,138 INFO    MainThread:3612948 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2025-05-18 12:22:42,138 INFO    MainThread:3612948 [wandb_run.py:_redirect():2161] Wrapping output streams.
2025-05-18 12:22:42,138 INFO    MainThread:3612948 [wandb_run.py:_redirect():2186] Redirects installed.
2025-05-18 12:22:42,140 INFO    MainThread:3612948 [wandb_init.py:init():847] run started, returning control to user process
